### ISSUE TITLE
TransitFeed 1.2.13 Release 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#344]
```
This issue captures all issues associated with the 1.2.13 release of 
transitfeed.
```

Original issue reported on code.google.com by `bdfer...@google.com` on 30 Jul 2012 at 2:17
- Blocked on: #345, #346, #347, #352, #358, #372, #377, #379

**Comment by GoogleCodeExporter** [comment]
Original comment by `bdfer...@google.com` on 21 Aug 2012 at 2:04
- Now blocked on: #345, #346

**Comment by GoogleCodeExporter** [comment]
Original comment by `bdfer...@google.com` on 2 Oct 2012 at 1:21
- Now blocked on: #347

**Comment by GoogleCodeExporter** [comment]
Original comment by `bdfer...@google.com` on 16 Nov 2012 at 2:57
- Now blocked on: #352

**Comment by GoogleCodeExporter** [comment]
Original comment by `bdfer...@google.com` on 19 Nov 2013 at 10:39
- Now blocked on: #372

**Comment by GoogleCodeExporter** [comment]
Original comment by `bdfer...@google.com` on 9 Sep 2014 at 1:03
- Now blocked on: #379

**Comment by GoogleCodeExporter** [comment]
Original comment by `bdfer...@google.com` on 9 Sep 2014 at 2:24
- Now blocked on: #358

**Comment by GoogleCodeExporter** [comment]
Original comment by `bdfer...@google.com` on 9 Sep 2014 at 2:57
- Now blocked on: #377

**Comment by GoogleCodeExporter** [comment]
Original comment by `bdfer...@google.com` on 26 Sep 2014 at 4:51
- Added labels: **Language-Python**

**Comment by GoogleCodeExporter** [comment]
```
Moved to https://github.com/google/transitfeed/issues/344
```

Original comment by `bdfer...@google.com` on 7 Oct 2014 at 7:39
- Changed state: **WontFix**
